tests/test_notifications.py

```
@@ -1032,3 +1050,209 @@ def test_pr_opened_channel_notification_skips_when_discord_mutations_disallowed(
+    dirty = "fix](https://evil.example) @everyone"
+    clean = _sanitize_discord_pr_title(dirty)
+    assert "\\]" in clean
+    assert "@everyone" not in clean
+    assert "@\u200beveryone" in clean
+
+    message = _build_pr_opened_channel_message(
+        ContributionEvent(
+            github_user="alice",
+            event_type="pr_opened",
+            repo="repo",
+            created_at=datetime.now(timezone.utc),
```

Ruff (0.16.0)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)